resources/partitioner/main.py

```diff
@@ -8,9 +8,31 @@
 import datetime
 import re
 import os
+import argparse
+import sys
 from pathlib import Path
 
 
+def get_session():
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        '--profile',
+        help='AWS profile from ~/.aws/credentials',
+        required=False,
+        default='default'
+    )
+    
+    args = parser.parse_args()
+
+    try:
+        session = boto3.Session(profile_name=args.profile)
```

Comment:
It looks like this over-rides the environment settings profile so you can't use this with aws-vault.  This should be fixable as was done in CloudMapper by using:
```
session_data = {}

    if args.profile_name:
        session_data["profile_name"] = args.profile_name

    session = boto3.Session(**session_data)
```

And also in `parser.add_argument` set the default to `None`

Reply:
See https://github.com/duo-labs/cloudmapper/blob/50365c31052b01f7dd9cd55e1ae6bd901ae846ae/commands/collect.py#L209